File: backend/JbdGjsWebService/Utils/StartFactory.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: 初期処理
'
' 作成日　　: 2024.07.17
' 作成者　　: AIPlus
' 変更履歴　:
' *******************************************************************

Namespace JBD.GJS.WebService
    Public Class StartFactory
        Public Shared Sub Start(ByVal configuration As Microsoft.Extensions.Configuration.ConfigurationManager)
            DaGlobal.GetConnectionStringFunc = Function(key) If(configuration.GetConnectionString(key), String.Empty)
            DaGlobal.GetSectionConfigStringValueFunc = Function(section, key) If(configuration.GetSection(section).GetValue(Of String)(key), String.Empty)
        End Sub
    End Class
End Namespace

File: backend/JbdGjsWebService/Utils/StartFactory.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: 初期処理
//
// 作成日　　: 2024.07.17
// 作成者　　: AIPlus
// 変更履歴　:
// *******************************************************************
using JbdGjsService.JBD.GJS.Db;

namespace JBD.GJS.WebService
{
    public class StartFactory
    {
        public static void Start(ConfigurationManager configuration)
        {
            DaGlobal.GetConnectionStringFunc = key => configuration.GetConnectionString(key) ?? string.Empty;
            DaGlobal.GetSectionConfigStringValueFunc = (section, key) => configuration.GetSection(section).GetValue<string>(key) ?? string.Empty;
        }
    }
}